tests/dummy/app/templates/docs/deprecations.md

```diff
@@ -13,41 +13,31 @@ Before:
 ```js
 export default class App extends Application {
   // ...
-  constructor() {
```

Comment:
why are you removing the `constructor` here? I can't see it as an improvement - it's only a different way to do that same thing.

Reply:
I don't understand how one can not see this as an improvement. Why would you want to use `constructor` and deliberately write more code than necessary if you can just use class fields? I'm pretty sure most people prefer class fields instead of overriding `constructor` and having to call `super` as well. Overriding `constructor` seems only useful if you want to do additional setup work or want to define fields based on others and the order matters, which is not the case here.

`constructor` isn't used here as well:
- https://ember-engines.com/docs/services (first example)
- https://ember-engines.com/docs/testing-unit (first example)
- https://ember-engines.com/docs/links#external-routes (first example)

Also, code that's part of the default blueprint for apps and addons uses class fields instead of `constructor`:
- https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/app/app.js#L7
- https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/app/router.js#L5
- ...

Reply:
That makes sense, thank you so much for bringing light to this @bertdeblock